lms/templates/sysadmin_dashboard.html

```diff
@@ -178,6 +181,31 @@ <h2 class="instructor-nav">
   <div class="form-actions">
 	<button type="submit" name="action" value="del_course">${_('Delete course from site')}</button>
   </div>
+  <hr />
+  <h4>Edit Course Tabs</h4>
+  <ul class="list-input">
+	  <li class="field text">
+		  <label for="course_id">
+			  ${_('Course ID')}:
+		  </label>
+		  <input type="text" name="course_id" style="width:60%"/>
```

Comment:
Inline styles can't be cached and are generally terrible for from a maintenance perspective; can we move this style to an external stylesheet?